internify/backend/routes/api/users.js

```diff
@@ -0,0 +1,72 @@
+var express = require("express");
+var router = express.Router();
+var UserData = require("../../models/UserData");
+
+//@route    GET api/users/getAll
+//@desc     Get All User Documents
+//@access   Public (?)
+
+router.get("/", function (req, res, next) {
```

Comment:
is this route `/users/`? If so, this is fine

Reply:
so technically if you added the router module in express backend, then leaving this as '/' is fine bc express will append the '/users' to it